react/features/toolbox/components/web/Toolbox.js

```diff
@@ -941,13 +1100,88 @@ class Toolbox extends Component<Props> {
         const overflowHasItems = Boolean(overflowMenuContent.filter(
             child => child).length);
         const toolbarAccLabel = 'toolbar.accessibilityLabel.moreActionsMenu';
+        const buttonsLeft = [];
+        const buttonsRight = [];
+
+        const maxNumberOfButtonsPerGroup = Math.floor(
+            (
+                this.state.windowWidth
+                    - 168 // the width of the central group by design
+                    - 48 // the minimum space between the button groups
+            )
+            / 56 // the width + padding of a button
+            / 2 // devide by the number of groups(left and right group)
```

Comment:
divide

Have you considered dynamically calculating? Might not be worth the hassle and code though.

Reply:
>  dynamically calculating

What do you mean?